src/app/domain/Part.ts

```diff
@@ -8,10 +8,12 @@ import { MGPOptional } from '../utils/MGPOptional';
 export interface Part extends FirestoreJSONObject {
     readonly typeGame: string; // the type of game
     readonly playerZero: MinimalUser; // the first player
+    readonly playerZeroElo: number, // the elo of the first player at the beginning of the part
     readonly turn: number; // -1 means the part has not started, 0 is the initial turn
     readonly result: IMGPResult;
 
     readonly playerOne?: MinimalUser; // the second player
+    readonly playerOneElo?: number, // must be modified/set whenever playerOne is modified/set
```

Comment:
modified or set? c'pas les deux !

Reply:
set the first time, modified the next ones